src/page-multi-facility/MultiFacilityImpactDashboard.tsx

```diff
@@ -198,7 +210,15 @@ const MultiFacilityImpactDashboard: React.FC = () => {
             </ScenarioTabList>
           </ScenarioTabs>
         </div>
-        {selectedTab === 0 && projectionsPanel}
+        {selectedTab == 0 && (
+          <ProjectionsPanel
+            facilitiesState={facilitiesState}
+            facilities={facilities}
+            localeDataSource={localeDataSource}
+            rtData={rtData}
+            handleFacilitySave={handleFacilitySave}
```

Comment:
`handleFacilitySave` is still getting re-created on every render, so memoizing probably isn't going to do anything with this component as-is; however, I think you should be able to wrap `handleFacilitySave` in a `useCallback` hook and stabilize it somewhat; I think its dependencies should be pretty stable.